### DISCUSSION TITLE
Is there any way to have different highlight colors for selected and normal elements?

### DISCUSSION THREAD

**Discussion by lominoss-git** [davatorium/rofi#1660] [Q&A] [answered]
I have the selected element's background color set to white and its foreground set to black, while a normal element's colors are reversed (bg: black, fg: gray). But the problem is that the highlighting colors are universal, meaning that both selected and normal elements get highlighted in the same way. Is there any way for me to have different highlighting colors? Thanks in advance :)

**Comment by DaveDavenport** [comment, marked as answer]
Not sure I follow.

Please check the `rofi-theme` manpage, the 'State' section.

https://github.com/davatorium/rofi/blob/next/doc/rofi-theme.5.markdown#state

**Reply by DaveDavenport** [reply]
```bash
echo -e "aap\naap2\naap3" | rofi -u 1,2 -dmenu -theme-str 'element-text normal.urgent { background-color: pink; highlight: strikethrough blue;}'
```
You see that highlight can be different per state.

**Reply by lominoss-git** [reply]
thank you so much it worked flawlessly
